woocommerce/woocommerce-gutenberg-products-block
# Create Cart i2 block

This PR creates Cart i2 block from Cart i1. I follow the same steps as Checkout i1 by copying, duplicating classes, and updating stuff to `cart-i2`.

Fixes #4717


### Testing
- run `composer install` to add the new class.
- Visit editor, try inserting Cart i2 block.
- Save page.
- Visit in on frontend, it should work.
